.gitignore

```diff
@@ -10,15 +10,6 @@
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
-# Package Files #
```

Comment:
why remove it 🤔

Reply:
Revert package files config in gitignore